src/coreclr/debug/ee/wks/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if (CLR_CMAKE_TARGET_WIN32)
 
   set(ASM_FILE ${CORDBEE_DIR}/${ARCH_SOURCES_DIR}/dbghelpers.asm)
 
-  if(CLR_CMAKE_HOST_ARCH_ARM OR CLR_CMAKE_HOST_ARCH_ARM64)
+  if(CLR_CMAKE_HOST_ARCH_ARM OR CLR_CMAKE_HOST_ARCH_ARM64 OR CLR_CMAKE_HOST_ARCH_LOONGARCH64)
```

Comment:
I think whole changes in this file can be moved to https://github.com/dotnet/runtime/pull/62886 to make this PR sef-contained.

Reply:
Thanks.
Had moved.